clarity/Cargo.toml

```diff
@@ -15,7 +15,7 @@ resolver = "2"
 
 [lib]
 name = "clarity"
-path = "./src/libclarity.rs"
+path = "./src/lib.rs"
```

Comment:
Without changing this the cargo mutants command would not see the content of the files and call the mutations for them

Reply:
Looks like a bug in `cargo-mutants`. I'd open an issue on their GH. It seems related to `cargo-mutants`' specific file recognition patterns. Rust typically looks for `src/lib.rs` as the main entry point for a crate, so unconventional filenames might not be detected by `cargo-mutants`. That's my guess.